A/armadillo/build_tarballs.jl

```diff
@@ -58,6 +60,7 @@ fi
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line.
 platforms = supported_platforms()
+platforms = [Linux(:x86_64)]
```

Comment:
?
```suggestion
```

Reply:
If this was to test locally a build only for a single platform, you may be interested in https://juliapackaging.github.io/BinaryBuilder.jl/dev/FAQ/#I-love-the-wizard,-but-now-I-want-to-break-free:-can-I-build-the-tarballs-without-it?-1  This is what I do all the time, without modifying the script

Reply:
Yeah, that's exactly what I need.  Thank you!  And this was an accidental commit. :(  Fixed in b6048ed, sorry about that.